ic-stable-structures/src/memory_mapped_files/memory_mapped_file.rs

```diff
@@ -170,7 +167,7 @@ mod tests {
     #[test]
     fn should_create_flush_memory_file() {
         with_temp_file(|path| {
-            let mut file_memory = MemoryMappedFile::new(path, true).unwrap();
+            let mut file_memory = MemoryMappedFile::new(path, PAGE_SIZE, true).unwrap();
```

Comment:
here, you are using the PAGE_SIZE as file size, I know that this is only a test, but this is confusing. Often people look at the tests to determine how to use something

Reply:
I've extracted DEFAULT_MAX_SIZE constant and added description why PAGE_SIZE is the minimal acceptable vallue